docs/ManualTests.md

```diff
@@ -302,11 +304,55 @@ Verify the following:
 11.	"Bin" Binary:
 *Expected: A B C D E F 2 3 4 5 6 7 8 9 are inactive. A maximum of 64 characters can be entered.*
 
+**Graphing Mode Test: Verify Graphing mode functions**
```

Comment:
You'll also want tests that target the error strings in KGF and variable editing. Those can come in another PR if you want.